app/controllers/invitations_controller.rb

```diff
@@ -62,7 +62,7 @@ def invitation_params
   end
 
   def sign_up_params
-    params.require(:user).permit(:name, :surname, :gender, :email, :password, :password_confirmation)
```

Comment:
Is it wrong? If you mean that it is other issue because I have problems with git